cmd/thanos/sidecar.go

```diff
@@ -46,7 +46,7 @@ func registerSidecar(m map[string]setupFunc, app *kingpin.Application, name stri
 	reloaderCfgFile := cmd.Flag("reloader.config-file", "Config file watched by the reloader.").
 		Default("").String()
 
-	reloaderCfgSubstFile := cmd.Flag("reloader.config-envsubst-file", "Output file for environment variable substituted config file.").
+	reloaderCfgOutputFile := cmd.Flag("reloader.config-output-file", "Output file for environment variable substituted config file.").
```

Comment:
Hah, actually I did not missed that I really liked that (: you are changing package fields that's all. We should avoid breaking unnecessary compatibility... Can we revert this flag rename? (: It is really not necessary.  

Sorry for the fuss - I should mentioned that earlier!

Reply:
reverted